bin/stop_be.sh

```diff
@@ -86,6 +86,20 @@ find "${UDF_RUNTIME_DIR}" -maxdepth 1 -name 'pyworker*' -print0 | while IFS= rea
 done
 
 
+# Stop profile collection daemon first
+profile_pidfile=$STARROCKS_HOME/bin/collect_be_profile.pid
+if [ -f $profile_pidfile ]; then
+    profile_pid=`cat $profile_pidfile`
+    if kill -0 $profile_pid > /dev/null 2>&1; then
+        echo "Stopping profile collection daemon (PID: $profile_pid)..."
+        kill -9 $profile_pid > /dev/null 2>&1
```

Comment:
may kill a wrong process if the collection daemon dead and the pid get reused by another process.